includes/admin/class-pinterest-for-woocommerce-admin.php

```diff
@@ -431,11 +455,22 @@ public function maybe_go_to_service_login_url() {
 				return false;
```

Comment:
I know this line isn't part of the set of changes, but the two `return` statements here are inconsistent. The first one does not return a value, while this one does. Since this method is being modified anyway, this value of `false` should be removed.